tcf_website/templates/course/course_professor.html

```diff
@@ -235,7 +247,8 @@ <h3 class="reviews-heading mb-0 mt-1 mr-4">{{ reviews|length }} {% if reviews|le
 {% block js %}
     <!-- For instructor data -->
     <script> // Create global js vars from template context vars
-       var courseID = "{{ course_id|escapejs }}";
+       var courseID = "{{ course.id|escapejs }}";
```

Comment:
Did you change back to `var` because global variables are necessary in this case

Reply:
Yep, otherwise the values couldn't be found from the js files.